build.cake

```diff
@@ -701,6 +701,18 @@ Task("RegisterUnity").Does(()=>
     // This will produce an error, but that's okay because the project "noproject" is used so that the
     // root isn't opened by unity, which could potentially remove important .meta files.
     ExecuteUnityCommand($"-serial {serialNumber} -username {username} -password {password}", "noproject");
+    var found = false;
+    if (Statics.Context.IsRunningOnUnix())
```

Comment:
Was this tested locally with local licenses?

Reply:
Yes